docs/workflow/README.md

```diff
@@ -35,6 +35,18 @@ For information about the different options available, supply the argument `--he
 build -h
 ```
 
+Alternatively, you can build and install runtime using [vcpkg](https://github.com/microsoft/vcpkg/) dependency manager:
```

Comment:
It might be worth rewording this a bit. `nethost` isn't the runtime itself but is instead part of the runtime hosting APIs: https://docs.microsoft.com/en-us/dotnet/core/tutorials/netcore-hosting

That is, it is a way for native applications to be able to self host the .NET Runtime in their own application independently of the normal host.